### ISSUE TITLE
Add Crash Reports API

### ISSUE THREAD

**Issue by dcramer** [getsentry/sentry#2519]
Probably in raven-js to start with:

```
Raven.showReportDialog(required string eventID, optional hash options)
```

Options would default to Raven's .config()'d values (for DSN, user context), but could be overwritten. ``routeError.jsx`` has our React implementation, and 500.html has our server side implementation of this.

**Comment by benvinegar** [comment]
Implemented in getsentry/raven-js#456 and scheduled for raven-js 2.1.0.

**Comment by benvinegar** [comment]
Added to Sentry here: 3663d3e7acdfc09fa13945e64547b2a4671b845b
